calc.rb

```diff
@@ -0,0 +1,65 @@
+# Brandi Wilson - CALCULATOR (Lunch)
+
+# PROMPTS THE USER.
+puts "What do you want to do, Master?"
+operator_prompt = "\tAddition\tSubtraction\n\tMultiplication\tDivision\n\tExponent\tModulo"
+puts "#{operator_prompt}"   # so the user knows what they can enter
+user_operator = gets.chomp.upcase   # captures user input
```

Comment:
Sweet. I noticed in your original version that you called `.upcase` multiple times, it good to see you caught that and made it so it only calls once.